NitroxPatcher/Patches/Dynamic/uGUI_PDA_SetTabs_Patch.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Collections.Generic;
+using System.Reflection;
+using HarmonyLib;
+using NitroxClient.GameLogic.HUD;
+using NitroxModel.Helper;
+
+namespace NitroxPatcher.Patches.Dynamic;
+
+public class uGUI_PDA_SetTabs_Patch : NitroxPatch, IDynamicPatch
+{
+    private readonly static MethodInfo TARGET_METHOD = Reflect.Method((uGUI_PDA t) => t.SetTabs(default));
+
+    public static bool Prefix(uGUI_PDA __instance, List<PDATab> tabs)
```

Comment:
If it is the same like in `uGUI_PDA_Initialize_Patch` (inserting code into base method) I would like to know why it's no transpiler and if it needs to be a Prefix I would like to have the same explaining and surrounding comments like in `uGUI_PDA_Initialize_Patch`.